articles/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Enabling HTTPS Engagement Tracking on SparkPost"
+description: "SparkPost supports HTTPS engagement tracking for customers via self-service for all SparkPost customers. To enable SSL engagement tracking for a domain, additional configuration for SSL keys is required."
+---
+
+## Overview
+
+SparkPost supports HTTPS engagement tracking for customers via self-service for all SparkPost customers. To enable SSL engagement tracking for a domain, additional configuration for SSL keys is required.
```

Comment:
The intro is a little odd and it doesn't call out the main mechanism in use: a CDN.  How about something like this:

`SparkPost supports HTTPS engagement tracking for all self-service customers. This article describes how to use a Content Delivery Network (CDN) to enable SSL engagement tracking for your domain.`